### PR TITLE
feat: Update styles in SCSS files meshery docs 

### LINKED ISSUE #10705: [Docs] Responsiveness issues
### Current State
https://docs.meshery.io/ currently in moblie view : 

     

it needs to be made responsive in smaller screen.

There were changes made to css recently, refer: https://github.com/meshery/meshery/commit/f660ca3c2e16795f4ec2fa977b8cfd2ef03a6da1

### Desired State

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by animeshchaudhri**
**Notes for Reviewers**

This PR fixes #10705

Updated styles in meshery/docs `alert.scss`, `elements.scss`, `mesheryctl.scss`, and `td-content.scss` to improve responsiveness and readability.

- Added media queries to adjust font sizes for headings (`h1`, `h2`, `h3`, `h4`) in `td-content.scss` for screens with a maximum width of 320px.
- Adjusted styles for `ul` and `ol` elements to make them responsive in `elements.scss` and added media queries .

# **Current State**
<div style="display: flex;">
  image
  image
</div>

# **After**
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 13ea0e9c0a564008b15da0f6e6281a40345ce6b3 at: https://6664d0c534f1033bfcd69165--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
@animeshchaudhri Thank you for your contribution.
Let's discuss this on Websites's call on Monday. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

File: docs/_sass/elements.scss [hunk 1]
```diff
@@ -149,3 +134,25 @@ button {
...
+
+  @media (max-width: 320px) {
+    font-size: 0.8rem;
+    padding-inline-start: 0.5rem;
+  }
+
+  & li {
+    margin: 2px 0;
+  }
+}
```
Comment (leecalcote) [inline docs/_sass/elements.scss:158]: No changes here, correct?

Including any changes outside of changes material to the issue at-hand makes PR review more challenging, risky, and comes with longer merge times. If you're able to remove these changes, that would be helpful.

File: docs/_sass/mesheryctl.scss [hunk 2]
```diff
@@ -24,9 +24,15 @@ div.prereqs {
     margin-top: 0;
   }
   color: #ddd;
-  margin: 20px;
+  margin: 0.5rem;
```
Comment (leecalcote) [inline docs/_sass/mesheryctl.scss:27]: Thank you for switching to `rem`.

File: docs/_sass/td-content.scss [hunk 3]
```diff
@@ -15,6 +15,25 @@
...
+    }
+
+    & > h3 {
+      font-size: 1.1rem; 
+      margin: 0.8rem 0 1rem; 
+    }
+    & > h4 {
+      font-size: 1rem; 
+    }
+  }
```
Comment (leecalcote) [inline docs/_sass/td-content.scss:36]: Good. 👍

File: docs/_sass/elements.scss (hunk 1 above)
Comment (animeshchaudhri) [reply docs/_sass/elements.scss:158]: The CSS was conflicting and not given priority , so I moved it down to ensure it applies correctly @leecalcote

**Comment by vishalvivekm** [comment]
Thank you @animeshchaudhri 
Let's discuss this in Meshery Dev meeting starting in 20 minutes.
Meeting minutes: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#
Meeting link: https://meet.layer5.io/meshery

**Comment by animeshchaudhri** [comment]
@vishalvivekm can you get this pr reviewed ?

**Comment by l5io** [comment]
🚀 Preview for commit 942e678b7b14b795b5ece3e30a9a62e7706b6ea5 at: https://666c4adb19c45a58ca93349b--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
@animeshchaudhri 
will you take a look at these picture of the same page in the preview of your PR and the live docs: 

#### Preview of these changes https://6664d0c534f1033bfcd69165--meshery-docs-preview.netlify.app/guides/infrastructure-management/overview: 



#### docs: https://docs.meshery.io/guides/infrastructure-management/overview:

**Comment by animeshchaudhri** [comment]
@vishalvivekm  will take a look

**Comment by vishalvivekm** [comment]
> @vishalvivekm will take a look

Thank you @animeshchaudhri, Let's discuss item on Websites call today, if you've made further progress.

**Pull request closed without merging**